### PR TITLE
[FIX,AUTOTVM] Fix printing of measure results

### PR DESCRIPTION
**Pull request by tkonolige**
The check for if the error_no was valid was wrong.

@masahi @jwfromm

### CODE REVIEW

**Comment by tkonolige** [comment]
@masahi Could you review?

File: python/tvm/autotvm/measure/measure.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class MeasureResult(namedtuple("MeasureResult", ["costs", "error_no", "all_cost"
     def __repr__(self):
         error_no_str = (
             str(self.error_no)
-            if self.error_no not in MeasureErrorNo
+            if not isinstance(self.error_no, (MeasureErrorNo, int))
```
Comment (AndrewZhaoLuo) [inline python/tvm/autotvm/measure/measure.py:59]: Can you rewrite this without a `not` in your check?

Also write a comment that explains what happens if it's neither an int or a MeasureErrorNo? (Is it just an exception?)

**Review by AndrewZhaoLuo — APPROVED** [review]

**Review by AndrewZhaoLuo — APPROVED** [review]

**Comment by AndrewZhaoLuo** [comment]
Looks like a scary error

**Comment by tkonolige** [comment]
It's caused by #10717, rebased.

**Pull request merged**